### DISCUSSION TITLE
http/congestion-control

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#236] [General]
# http/congestion-control

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/http/congestion-control.html

**Comment by useforgithub** [comment]
2.6.2 早期拥塞控制旨在收敛
有一个错别字：当发送“发”开始发送数据时

**Reply by isno** [reply]
谢谢，已纠正。

**Comment by zjyyzhengli** [comment]
1. 图 2-22 中的两个黑色圆圈即代表网络传输的最优点 --> 这个图并未看到，抑或是图片编号错误？
2. 测量最大带宽时，必须填满瓶颈链路，此时缓冲区被~~沾~~**占**满

**Reply by isno** [reply]
1. 图 2-22 有的，在文章最开始的位置
2. 已修复

谢谢纠正。

**Comment by WYXNB** [comment]
图 2-22 中的两个黑色圆圈即代表网络传输的最优点。应该是图2-19吧？

**Reply by isno** [reply]
这而是写错了，可以改成 图 2-21
